README.md

```diff
@@ -1,2 +1,45 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+int MAX_GUESSES = 4
+string TO_GUESS = "Word to guess"
+bool won = False
+print("Guess the " + TO_GUESS.length() + " letter word")
+print("_" * TO_GUESS)
+for(int y = 0; y < MAX_GUESSES; y++)
+{
+  int counter = 0
+  bool chk = True
+  while(chk)
+  {
+    string ans = input()
+    if (ans.length() != TO_GUESS.length())
+    {
+      print("Please enter a " + TO_GUESS.length() + "word")
+      chk = False
+    }
+  }
+  for(int x = 0; x < ans.length(); x++)
```

Comment:
The worksheet asked you to provide pseudocode specifically for measuring the similarity score between the words, which you have done here -- however you did not need to provide pseudocode for the rest of the game. Be sure to read the specifications carefully in future to avoid doing extra unnecessary work!